ISHLogDNA.xcodeproj/project.pbxproj

```diff
@@ -166,10 +217,14 @@
 		DC0C28071EF0764500B27023 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 0830;
+				LastSwiftUpdateCheck = 0900;
```

Comment:
Works fine with Xcode 8.

Also, this only affects sample projects and tests. We don't provide a framework project, just two source files.